use-ci-nexus-cache/action.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Use CI Nexus cache
+
+description: Sets up use of co-located pull-through cache for Maven artifacts via ~/.m2/settings.xml.
+
+inputs:
+  server_username:
+    description: 'Nexus Username'
+  server_password:
+    description: 'Nexus Password'
+  server_id:
+    description: 'server ID'
+
+runs:
+  using: composite
+  steps:
+  - name: Create ~/.m2/settings.xml with cache configuration
+    shell: bash
+    run: |
+      cat <<EOF > ~/.m2/settings.xml
+      <?xml version="1.0" encoding="UTF-8"?>
+      <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+        <interactiveMode>false</interactiveMode>
+        <servers>
+          <server>
+            <id>${{ inputs.server_id }}</id>
```

Comment:
what is `server_id`? does it have to be dynamic?

Reply:
this is to make the action more generic, it could be that some teams are not using `camunda-nexus` as server id. maybe just `nexus` or `camunda_nexus`

Reply:
makes sense, thanks!